rust/connlib/libs/client/src/control.rs

```diff
@@ -254,4 +255,8 @@ impl<CB: Callbacks + 'static> ControlSession<Messages, CB> for ControlPlane<CB>
     fn socket_path() -> &'static str {
         "device"
     }
+
+    fn retry_strategy() -> ExponentialBackoff {
+        ExponentialBackoffBuilder::default().build()
```

Comment:
What the timeouts will be with default exponential backoff? We need to have to limit the max value to make sure the connection is getting recovered faster after a long-ish downtime.

Reply:
For the clients(this file) the limit to the max interval by default is 1 minute, I felt that was fine and max retry time is 15 minutes.

For gateways, the difference is the max retry time was updated to None(so it will retry forever).